### PR TITLE
{OP-66} Atomically update votes

### PR DESCRIPTION
**Pull request by vkurup**

### CODE REVIEW

**Comment by ejucovy** [comment]
Wow, I sure wish I'd known about this `F` thing last January when I was updating vote tallies by hand for the popular submissions :-)

**Comment by vkurup** [comment]
Heh.... Django has so many hidden gems :)

File: opendebates/views.py [hunk 1]
```diff
@@ -236,8 +237,10 @@ def vote(request, id):
         )
     )
     if created:
+        # update the DB with the real tally
+        Submission.objects.filter(id=id, approved=True).update(votes=F('votes')+1)
```
Comment (dpoirier) [inline opendebates/views.py]: Do we need to include `approved=True` here?  ID is pretty specific :-)

**Comment by dpoirier** [comment]
One comment and :shipit:

I started to do this yesterday when working on this view but saw we needed to give the client the new count. It didn't occur to me that we didn't have to give the client the exact count :-)

**Pull request merged**